test/hex_tarball_tests.erl

```diff
@@ -17,6 +17,7 @@ memory_test() ->
 disk_test() ->
     in_tmp(fun() ->
         ok = file:write_file("foo.erl", <<"-module(foo).">>),
+        ok = file:change_mode("foo.erl", 8#644),
```

Comment:
let's use `8#100644` like everywhere else

Reply:
Those are completely different numbers though.

Reply:
yes, obviously. Is it the case that `file:change_mode` takes 8#644 but `file:write_file_info` would take 8#100644 and they'd mean the same thing? in that case sounds good as is.

Reply:
Yeah I'm not sure what those extra bits are for. It does not make any difference for the test.